3.Tests/UI/StepDefinitions/Settings/Profile/UpdateFullNameStepDefinition.cs

```diff
@@ -0,0 +1,69 @@
+﻿using OpenQA.Selenium;
+using SeleniumTest.Core.Drivers;
+using SeleniumTest.Core.Interfaces;
+using SeleniumTest.Tests.Steps.Commons;
+using TechTalk.SpecFlow;
+using Views.WebAppPages;
+
+namespace SeleniumTest.Tests;
+
+[Binding]
+[Scope(Feature = "Full Name update")]
+[TestFixture]
+public class UpdateFullNameStepDefinition : CommonSteps
+{
+    private readonly ProfilePage _profile;
+    private readonly IGenericWebDriver _driver;
+    private readonly ScenarioContext _scenarioContext;
+
+    public UpdateFullNameStepDefinition(ScenarioContext scenarioContext, ChromeWebDriver driver) : base(scenarioContext, driver)
+    {
+        _scenarioContext = scenarioContext;
+        _driver = driver;
+        _profile = new ProfilePage(driver);
+    }
+
+    [Given(@"the user clicks on the Settings option on the Nav Bar")]
+    public void GiventheuserclicksontheSettingsoptionontheNavBar()
+    {
+        _profile.Settings.Click();
+        IWebElement settingsDialog = _driver.Instance().FindElement(By.XPath("//div[@id='settingsDialog']"));
```

Comment:
We should avoid using the webdriver directly on the test layer of our framework, we have to leave that responsibility to the Pages layer, actually the UIElements layer even

Reply:
Fixed.